src/common/SurgeStorage.h

```diff
@@ -1344,6 +1344,8 @@ class alignas(16) SurgeStorage
     bool oscReceiving{false};
     bool oscSending{false};
 
+    int voiceCount;
+
```

Comment:
I'm kind of bummed we have to put voiceCount onto SurgeStorage. Is there some reason it can't live on SurgeSynthesizer instead, where it more naturally lives? Storage is more 'static' state than 'runtime' state.

If we move it to Synth, what changes? Do you not get access to it at setup time or something?

If the problem is SLFO don't have a Synth reference, we can fix that probably.

This might be what we have to do though; but if it is what we have to do then (1) we should probably make it `std::atomic<int32_t>` and (2) we should add a comment that its somewhat dissatisfying and why we added i there.

Reply:
Was not too happy with it either but was the only way I got it working so far, was having trouble passing that atomic int as a pointer and doing it static it won't update with voice changes since it only gets passed to SurgeVoice() on creation.

Tried adding a `this` pointer from SurgeSynthesizer to pass to SurgeVoice() and from there to valueAt() in FormulaModulationHelper.cpp where it is needed but could not get that working, ideally we would use it like SurgeGUIEditor.cpp [here](https://github.com/surge-synthesizer/surge/blob/af631b506566184f438f5dfeb24c7acb7d8e6ed7/src/surge-xt/gui/SurgeGUIEditor.cpp#L797) but adding the SurgeSynth class reference seemed beyond my C++ skills and this seemed like a smaller change at that point. So yeah, I'm all for leaving SurgeStorage as is if there is a better way.

Reply:
OK for now lets add it then and put a comment and open an issue to ponder a better spot.

It's not entirely wrong. But it's also not entirely right. But I'm not sure where else it could go right now.